src/config/dynamic-theme-fixes.config

```diff
@@ -6712,77 +6712,56 @@ INVERT
 drive.google.com
 
 INVERT
-img[src$="google_gsuite"]
-div[role="menuitem"] svg
-div[role="dialog"][style^="width: 350px;"] div[role="button"][tabindex="0"]
-div[role="dialog"] ~ div[role="menu"] > div[role="menuitem"] > div > div:not([style*="background-image"])
-img[src$="type/audio/mp3"]
-div[role="listitem"] > div > div > div > svg[fill="#000000"] > path
-div[role="menu"] > div > div[role="menuitem"] > div > div > div
-div[data-label="nd"] > div > div > svg > path[fill="#000000"]
-div[role="document"] > div[role="button"] .a-b-c
-div > svg > circle[fill="white"]
-img[src*="empty_state_trash"]
+div[style*="add_black"]
+div[aria-label="Show filter toolbar"]
+div.settings-view-general div[style]:last-child
+nav div[style]:last-child:not(div[role="group"])
 
 CSS
-span[data-type="spelling"] {
-    mix-blend-mode: normal !important;
+* {
```

Comment:
Did you try `:root` instead of `*`? Custom CSS properties are inherited down the elements tree.

Reply:
I tried it, I use `*` because the color is set under a unique id, also I can't guarantee that id is the same for everyone

![image](https://github.com/darkreader/darkreader/assets/87893636/4d1b0597-cc18-4467-9d93-0030c11c3e7c)